iotdb-client/client-py/iotdb/utils/Field.py

```diff
@@ -38,13 +38,19 @@ def copy(field):
                 output.set_bool_value(field.get_bool_value())
             elif output.get_data_type() == TSDataType.INT32:
                 output.set_int_value(field.get_int_value())
-            elif output.get_data_type() == TSDataType.INT64:
+            elif (
+                output.get_data_type() == TSDataType.INT64
+                or output.get_data_type() == TSDataType.TIMESTAMP
+            ):
                 output.set_long_value(field.get_long_value())
             elif output.get_data_type() == TSDataType.FLOAT:
                 output.set_float_value(field.get_float_value())
             elif output.get_data_type() == TSDataType.DOUBLE:
                 output.set_double_value(field.get_double_value())
-            elif output.get_data_type() == TSDataType.TEXT:
+            elif (
+                output.get_data_type() == TSDataType.TEXT
+                or output.get_data_type() == TSDataType.STRING
+            ):
                 output.set_binary_value(field.get_binary_value())
             else:
```

Comment:
Should DATE and BLOB be processed here?

Reply:
This pr is focus on the table model, I think we need another pr to completely support the new data types.